src/asset-relocator.js

```diff
@@ -934,19 +934,21 @@ module.exports = async function (content, map) {
               }
             break;
             case NODE_GYP_BUILD:
-              if (node.arguments.length === 1 && node.arguments[0].type === 'Identifier' &&
-                  node.arguments[0].name === '__dirname' && knownBindings.__dirname.shadowDepth === 0) {
-                transformed = true;
-                let resolved;
-                try {
-                  resolved = nodeGypBuild.path(dir);
-                }
-                catch (e) {}
-                if (resolved) {
-                  staticChildValue = { value: resolved };
-                  staticChildNode = node;
-                  emitStaticChildAsset(path);
-                  return backtrack(this, parent);
+              if (node.arguments.length) {
```

Comment:
nit: prefer explicit conditional check here like `length > 0` instead of implicit truthy check

Reply:
Updated 👍 (and updated the place I copied it from 😅 )